Tools/CISupport/build-webkit-org/steps_unittest.py

```diff
@@ -726,78 +726,142 @@ def test_lot_of_failures(self):
 
-    def test_success(self):
+    def configureStep(self):
         self.setupStep(RunLLDBWebKitTests())
         self.setProperty('configuration', 'release')
+        self.setProperty('buildername', 'LLDBWebKit-Tests-EWS')
+        self.setProperty('buildnumber', '101')
+        self.setProperty('workername', 'ews100')
+
+    def test_success(self):
+        self.configureStep()
         self.expectRemoteCommands(
             ExpectShell(
                 workdir='wkdir',
                 timeout=1200,
                 logEnviron=True,
-                command=['python3', 'Tools/Scripts/test-lldb-webkit', '--verbose', '--no-build', '--release'],
+                command=[
+                    'python3',
+                    'Tools/Scripts/test-lldb-webkit',
+                    '--verbose',
+                    '--no-build',
+                    '--release',
+                    '--suite', 'lldb-webkit-test',
+                    '--buildbot-master', DNS_NAME,
+                    '--builder-name', 'LLDBWebKit-Tests-EWS',
+                    '--build-number', '101',
+                    '--buildbot-worker', 'ews100',
+                    '--report', RESULTS_WEBKIT_URL,
+                ],
             ) + 0,
         )
         self.expectOutcome(result=SUCCESS, state_string='lldb-webkit-test')
         return self.runStep()
 
     def test_unexpected_failure(self):
-        self.setupStep(RunLLDBWebKitTests())
-        self.setProperty('configuration', 'release')
+        self.configureStep()
         self.expectRemoteCommands(
             ExpectShell(
                 workdir='wkdir',
                 timeout=1200,
                 logEnviron=True,
-                command=['python3', 'Tools/Scripts/test-lldb-webkit', '--verbose', '--no-build', '--release'],
+                command=[
+                    'python3',
+                    'Tools/Scripts/test-lldb-webkit',
+                    '--verbose',
+                    '--no-build',
+                    '--release',
+                    '--suite', 'lldb-webkit-test',
+                    '--buildbot-master', DNS_NAME,
+                    '--builder-name', 'LLDBWebKit-Tests-EWS',
+                    '--build-number', '101',
+                    '--buildbot-worker', 'ews100',
+                    '--report', RESULTS_WEBKIT_URL,
+                ],
             ) + 2,
         )
         self.expectOutcome(result=FAILURE, state_string='lldb-webkit-test (failure)')
         return self.runStep()
 
     def test_failure(self):
-        self.setupStep(RunLLDBWebKitTests())
-        self.setProperty('configuration', 'release')
+        self.configureStep()
         self.expectRemoteCommands(
             ExpectShell(
                 workdir='wkdir',
                 timeout=1200,
                 logEnviron=True,
-                command=['python3', 'Tools/Scripts/test-lldb-webkit', '--verbose', '--no-build', '--release'],
+                command=[
```

Comment:
Oh, you should also probably be adding `env={'RESULTS_SERVER_API_KEY': 'test-api-key'}` in `ExpectShell` so unit tests are accurate.